src/main/java/plugily/projects/villagedefense/arena/managers/spawner/SimpleZombieSpawner.java

```diff
@@ -109,7 +109,10 @@ default int getSpawnWeight(Arena arena, int wave, int phase, int spawnAmount) {
    * @param arena    the arena
    */
   default void spawnZombie(Location location, Arena arena) {
-    Zombie zombie = spawnZombie(location);
+    Creature zombie = spawnZombie(location);
+    if (zombie == null) {
+      return;
+    }
```

Comment:
Where and when can the `spawnZombie(Location)` method be null?

Reply:
flag that as Nullable

Reply:
Doesn't seem like any of these methods can return null.

Reply:
My MythicMobs addon use this spawner but the user can somehow set the mob that's not a `Creature`.
And this is to make sure for all other future addons.